layouts/partials/page/script.html

```diff
@@ -1,3 +1,22 @@
+<script defer src="/presidium.js"></script>
+{{ if $.Site.Params.enterprise_enabled }}
+<script defer src="{{ $.Site.Params.enterprise_script }}"></script>
+<script>
+  const toolbarElement = document.querySelector('#presidium-enterprise');
+  const navigationElement = document.querySelector('#presidium-navigation');
+  if (toolbarElement) {
+      const toolbarHeight = toolbarElement.offsetHeight;
+      if (toolbarHeight) {
+          navigationElement.style.top = `${toolbarHeight}px`;
+      }
+  }
+</script>
+{{ end }}
+
+<script defer src="{{ "assets/scripts/jquery.js" | absURL }}"></script>
+<script defer src="{{ "assets/scripts/bootstrap.js" | absURL }}"></script>
+<script defer src={{ "assets/scripts/mermaid.js" | absURL }}></script>
+
 <script>
```

Comment:
Seeing that we're deferring the external jquery script, would we not run in to an issue where this internal script block executes before the JQuery external dependency has executed?  Previously this would have worked because all script blocks were blocking.

Reply:
No, deferred scripts keep their relative order, just like regular scripts. `async` doesn't.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#defer
> Scripts with the defer attribute will execute in the order in which they appear in the document.

Reply:
But the inline script is not deferred. Or is it deferred by default?

Reply:
You are right. I added `type="module"` to all the inline scripts. It has the same effect as `defer`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#module
> This value causes the code to be treated as a JavaScript module. The processing of the script contents is deferred. The charset and defer attributes have no effect.

Reply:
![Screenshot 2023-10-10 at 09 45 48](https://github.com/SPANDigital/presidium-theme-website/assets/48946187/d8c93586-26e5-43f2-9141-2aa77aa4d0c3)

Reply:
Ahhhh nice!